csrc/options.cpp

```diff
@@ -166,6 +166,7 @@ const std::unordered_map<std::string, EnableOption>& getEnableOptions() {
           {"static_fusion_count", EnableOption::StaticFusionCount},
           {"wait_debugger", EnableOption::WaitDebugger},
           {"warn_register_spill", EnableOption::WarnRegisterSpill},
+          {"ws_persistent", EnableOption::WarpSpecializedPersistent},
```

Comment:
This flag is ambiguous because we already have warp-specialized persistent matmul kernels.

Reply:
revised to `{"ws_normalization", EnableOption::WarpSpecializedNormalization}`, this is a tmp flag and will be removed after the whole development is done.